api/api/test/test_signals.py

```diff
@@ -0,0 +1,274 @@
+    )
+    await asyncio.sleep(1)
+    task.cancel()
+
+    client_session_get_mock.assert_called()
+
+    assert (
+        application_mock_with_installation_uid['update_information']['status_code']
+        == status
+    )
+    available_update = application_mock_with_installation_uid['update_information'][
+        'available_update'
+    ]
+
+    if len(patch):
+        assert available_update == response_data['data']['patch'][0]
+    elif len(minor):
+        assert available_update == response_data['data']['minor'][0]
+    elif len(major):
+        assert available_update == response_data['data']['mayor'][0]
+
+
+async def test_get_update_information_injects_correct_data_into_app_context_when_satatus_not_200(
+    application_mock_with_installation_uid, client_session_get_mock
+):
+    response_data = {'errors': {'detail': 'Unathorized'}}
+    status = 403
+
+    client_session_get_mock.return_value.__aenter__.return_value.status = status
+    client_session_get_mock.return_value.__aenter__.return_value.json.return_value = (
+        response_data
+    )
+    task = asyncio.create_task(
+        get_update_information(application_mock_with_installation_uid)
+    )
+    await asyncio.sleep(1)
+    task.cancel()
+
+    client_session_get_mock.assert_called()
+    assert (
+        application_mock_with_installation_uid['update_information']['status_code']
+        == status
+    )
+    assert (
+        application_mock_with_installation_uid['update_information']['message']
+        == response_data['errors']['detail']
+    )
+
+
+@pytest.mark.asyncio
+async def test_get_update_information_make_request_with_correct_parameters(
+    application_mock_with_installation_uid, client_session_get_mock
+):
+    version = '4.8.0'
+    with patch('api.signals.wazuh.__version__', version):
+        task = asyncio.create_task(
+            get_update_information(application_mock_with_installation_uid)
+        )
+        await asyncio.sleep(1)
+        task.cancel()
+
+        client_session_get_mock.assert_called()
+
+        client_session_get_mock.assert_called_with(
+            RELEASE_UPDATES_URL,
+            headers={
+                WAZUH_UID_KEY: application_mock_with_installation_uid[INSTALLATION_UID_KEY],
+                WAZUH_TAG_KEY: f'v{version}',
+            },
+        )
+
+
+@pytest.mark.asyncio
+async def test_get_update_information_sleep_correctly_until_next_excecution(
+    application_mock_with_installation_uid, client_session_get_mock
+):
+    with patch('api.signals.asyncio') as sleep_mock:
+        task = asyncio.create_task(
+            get_update_information(application_mock_with_installation_uid)
+        )
+        await asyncio.sleep(1)
+        task.cancel()
+
+        client_session_get_mock.assert_called()
+        sleep_mock.sleep.assert_called_with(ONE_DAY_SLEEP)
+
+
+@pytest.mark.parametrize(
+    'cconfig,update_check_config,registered_tasks',
```

Comment:
Not sure if it was a typo or on purpose, but I would prefer `config` or `cluster_config`. The same applies to the ones below

```suggestion
    'config,update_check_config,registered_tasks',
```